rpc/src/commands/chains/blocks/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub mod get_balance;
 pub mod get_blocks_in_chain;
 pub mod get_invalid_blocks_in_chain;
+use crate::responses::chains::blocks as block_responses;
```

Comment:
I couldn't stand the constant `use crate::responses::chains::blocks::`, so now they're just `use super::block_responses::`.